mllib/src/main/scala/org/apache/spark/ml/tuning/CrossValidator.scala

```diff
@@ -56,7 +56,10 @@ private[ml] trait CrossValidatorParams extends ValidatorParams {
 
 /**
  * :: Experimental ::
- * K-fold cross validation.
+ * CrossValidator begins by splitting the dataset into a set of non-overlapping randomly
```

Comment:
I like the improved description, but can we please keep the phrase "k-fold cross validation?"  It's a very common phrase and will be useful for people using keyword search.  Thanks!